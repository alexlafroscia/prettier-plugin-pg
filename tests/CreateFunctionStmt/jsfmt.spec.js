run_spec(__dirname, ["postgresql"]);
