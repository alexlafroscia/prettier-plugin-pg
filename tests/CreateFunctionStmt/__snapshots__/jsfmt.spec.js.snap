// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple.sql - postgresql-verify: simple.sql 1`] = `
create function foo() returns text as $$ select 'hi'; $$ language sql;
create function foo() returns text language plv8 as $$ var a = 1; var b = 2; return a + " + " + b+"="+(((a + b))); $$;
create function foo() returns text language plv8 as $$
var a = 1;
  var b = 2;
  return a + " + " +
b+"="+(((a + b)));
$$;
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
CREATE FUNCTION foo() RETURNS text LANGUAGE 'sql' AS $$
  SELECT 'hi';
$$;
CREATE FUNCTION foo() RETURNS text LANGUAGE 'plv8' AS $$
  var a = 1;
  var b = 2;
  return a + " + " + b + "=" + (a + b);

$$;
CREATE FUNCTION foo() RETURNS text LANGUAGE 'plv8' AS $$
  var a = 1;
  var b = 2;
  return a + " + " + b + "=" + (a + b);

$$;
`;
